db/views.h

```diff
@@ -59,6 +59,24 @@ typedef struct timepart_sc_arg {
     void *tran; /*remove?*/
 } timepart_sc_arg_t;
 
+
+enum systable_columns {
+    VIEWS_NAME
+    ,VIEWS_PERIOD
+    ,VIEWS_SHARDNAME=VIEWS_PERIOD
```

Comment:
All this to save one extra enum definition? :) Please separate them.

Reply:
and move them to timepartitions.c.

Reply:
See above.

Reply:
Test coming.